collector/loadavg.go

```diff
@@ -3,18 +3,15 @@
 package collector
 
 import (
+	"errors"
 	"fmt"
-	"io/ioutil"
-	"strconv"
-	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/log"
 )
 
-const (
-	procLoad = "/proc/loadavg"
-)
+// #include <stdlib.h>
```

Comment:
We just went through some effort to get node exporter free of cgo (at least for the Linux parts). If FreeBSD requires cgo to get the loadavg, I would prefer splitting this into two collectors, one for Linux and one for FreeBSD, where the former doesn't need cgo.